.storybook/config.js

```diff
@@ -1,10 +1,14 @@
-import 'font-awesome/css/font-awesome.min.css'
 import 'tachyons-egghead'
 import {configure, setAddon} from '@kadira/storybook'
-import infoAddon from '@kadira/react-storybook-addon-info';
+import infoAddon from 'react-storybook-addon-info';
+import withPropsCombinations, {setDefaults} from 'react-storybook-addon-props-combinations'
+
 setAddon(infoAddon)
-import withPropsCombinations, { setDefaults } from 'react-storybook-addon-props-combinations'
 setAddon(withPropsCombinations)
+setDefaults({
+  showSource: false,
+  mustProvideAllProps: true,
+})
```

Comment:
This sets up the prop combinations and requires all props to be passed so that the styleguide is accurate